media-plugins/qmmp-plugin-pack/qmmp-plugin-pack-2.1.2.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake flag-o-matic
+
+DESCRIPTION="Set of extra plugins for Qmmp"
+HOMEPAGE="https://qmmp.ylsoftware.com/"
+SRC_URI="https://qmmp.ylsoftware.com/files/${PN}/$(ver_cut 1-2)/${P}.tar.bz2"
+
+LICENSE="GPL-2"
+SLOT="0/6"
```

Comment:
I could not make sense of this change and alas it was not mentioned in git message, so also dropping from merge.